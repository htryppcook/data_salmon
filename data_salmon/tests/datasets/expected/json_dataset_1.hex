616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff616263640000ffff