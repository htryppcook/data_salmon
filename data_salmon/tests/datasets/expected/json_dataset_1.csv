field0_length,field0,field1
1,0,abcd
1,1,abcd
1,2,abcd
1,3,abcd
1,4,abcd
1,5,abcd
1,6,abcd
1,7,abcd
1,8,abcd
1,9,abcd
